src/Symfony/Bundle/FrameworkBundle/Resources/config/assets.php

```diff
@@ -30,8 +30,8 @@
     $container->services()
         ->set('assets.packages', Packages::class)
             ->args([
-                service('assets.empty_package'),
-                [],
+                service('assets._default_package'),
+                tagged_iterator('assets.package', 'package', 'getDefaultName'),
```

Comment:
we should remove `getDefaultName` and let the default happen, which is equivalent to `getDefaultPackageName`

Reply:
Thanks for the review. I didn't know about this default value.